test/e2e/service.go

```diff
@@ -192,9 +192,12 @@ var _ = Describe("Services", func() {
 		Expect(foundRO).To(Equal(true))
 	})
 
-	It("should serve a basic endpoint from pods", func(done Done) {
+	It("should serve a basic endpoint from pods", func() {
+
 		serviceName := "endpoint-test2"
-		ns := api.NamespaceDefault
+		ns_, err := createTestingNS(namespace0, c)
```

Comment:
I would prefer you to use namespace0 directly here.
Either in this PR, or in a separate one, you could update BeforeEach() to call createTestingNS() instead of the clunky timestamp that it currently uses (I wrote it, so I'm allowed to call it clunky :-).